examples/deferred_processing/README.md

```diff
@@ -0,0 +1,340 @@
+    ```
+
+### Transactions
+
+15. If you want to run a particular set of data in `images_dp_2` 
+    against a particular branch of `edges_dp`,
+    you need to perform two operations
+    - commit data to `images_dp_2` and
+    - point `edges_dp@master` to the branch of interest.
+    
+    This will result in two jobs being triggered.
+    - `images_dp_2@master` running against whatever is currently in `edges_dp@master`
+    - `images_dp_2@master` running against whatever you set `edges_dp@master` to
+    
+    Remember that in step 14 above, 
+    we performed the `create branch` operation.
+    Now we perform the commit
+    and see that another job got triggered.
+    
+    ```sh
+    $ pachctl put file images_dp_2@master:3Kr6Mr6.jpg  -f http://imgur.com/3Kr6Mr6.jpg
+    $ pachctl list job
+    ID                               PIPELINE           STARTED            DURATION  RESTART PROGRESS  DL       UL       STATE   
+    1c8d2bb48bca451ba43fdc5308a24d6d montage_dp         9 seconds ago      4 seconds 0       1 + 0 / 1 770.7KiB 906.7KiB success 
+    01bb8e400c054a18a5c2c691f0b5a65d montage_dp         About a minute ago 2 seconds 0       0 + 1 / 1 0B       0B       success 
+    96439afdaf8249be81773c89fc87b366 edges_dp           About a minute ago 3 seconds 0       1 + 4 / 5 204KiB   0B       success 
+    4b05f2c0ddf14ab0a4ca8f4ee2d6d9ff montage_dp         3 minutes ago      3 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    1a8d2fefc2ba450e9102ffd6b74e503d montage_dp         4 minutes ago      4 seconds 0       1 + 0 / 1 693.8KiB 673.6KiB success 
+    c5feebe5c6ae48e0adc4f5bd7be9a9ac edges_dp           4 minutes ago      3 seconds 0       1 + 3 / 4 175.1KiB 0B       success 
+    7d6aa9bbe1484e2e9ac60a7f84147e1b montage_dp         25 minutes ago     1 second  0       0 + 0 / 0 0B       0B       success 
+    f7a7d71dd7e74a3f9c48d4b9d2897ba3 edges_dp           25 minutes ago     4 seconds 0       2 + 1 / 3 181.1KiB 0B       success 
+    f9c620d1a4cc421a9c3923b749e9e6ed edges_dp           25 minutes ago     3 seconds 0       1 + 0 / 1 57.27KiB 0B       success 
+    ```
+    
+16. If you want to just have one job 
+    where `images_dp_2@master` runs against whatever you set `edges_dp@master` to,
+    you can use Pachyderm transactions.
+    First step is to start a transaction.
+    
+    ```sh
+    $ pachctl start transaction
+    Started new transaction: 018b9e52-e483-4a0a-accd-cef2656f7b8c
+    ```
+    
+17. Once the transaction is started,
+    you start all commits and branch creations 
+    within the scope of the transaction. 
+    
+    ```sh
+    $ pachctl start commit  images_dp_2@master
+    Added to transaction: 018b9e52-e483-4a0a-accd-cef2656f7b8c
+    e6e68bd54b9049658fca31aa7e905fcd
+    $ pachctl create branch edges_dp@master --head stable_1_1
+    Added to transaction: 018b9e52-e483-4a0a-accd-cef2656f7b8c
+    ```
+
+
+18. Before you put any files in an repos, 
```

Comment:
funky question:

What happens if after this transaction, the edges@master branch is moved again by someone else. Once the transaction commit finishes, presumably the job will run against whatever master is pointing to at that time, right? Or will it be against the specific commit that was `master` during the transaction?

Reply:
I'm going to run another test on this after I complete this PR, because that's a question that's out of the scope of this doc. :-)